### PR TITLE
Fix special characters in Wi-Fi SSID

### PR DESCRIPTION
**Pull request by salvatore-coppola**
Now UTF--8 encoded character are supported in SSID.
The SSID is converted in its hex representation in wpa_supplicant files.

### CODE REVIEW

**Comment by MMaiero** [comment]
@mstankovic @marcellorinaldo Can you have a review of this PR, please?

File: kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/network/TabWirelessUi.java [hunk 1]
```diff
@@ -1232,7 +1232,7 @@ private void initSsid() {
 
             @Override
             public String getValue(GwtWifiHotspotEntry object) {
```
Comment (marcellorinaldo) [inline kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/network/TabWirelessUi.java:1234]: Update copyright header

File: kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/server/GwtNetworkServiceImpl.java [hunk 2]
```diff
@@ -1018,7 +1017,7 @@ public void updateNetInterfaceConfigurations(GwtXSRFToken xsrfToken, GwtNetInter
             List<WifiHotspotInfo> wifiHotspotInfoList = nas.getWifiHotspotList(interfaceName);
             if (wifiHotspotInfoList != null) {
                 for (WifiHotspotInfo wifiHotspotInfo : wifiHotspotInfoList) {
-                    String ssid = GwtSafeHtmlUtils.htmlEscape(wifiHotspotInfo.getSsid());
+                    String ssid = wifiHotspotInfo.getSsid();
```
Comment (marcellorinaldo) [inline kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/server/GwtNetworkServiceImpl.java:1020]: Update copyright header

File: kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/visitor/linux/WpaSupplicantConfigWriter.java [hunk 3]
```diff
@@ -45,6 +45,7 @@
 import org.eclipse.kura.net.wifi.WifiConfig;
 import org.eclipse.kura.net.wifi.WifiMode;
 import org.eclipse.kura.net.wifi.WifiSecurity;
+import org.eclipse.kura.util.base.StringUtil;
```
Comment (marcellorinaldo) [inline kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/visitor/linux/WpaSupplicantConfigWriter.java:48]: Update copyright header

File: kura/org.eclipse.kura.linux.net/src/main/java/org/eclipse/kura/linux/net/util/IWAPParser.java [hunk 4]
```diff
@@ -21,6 +21,7 @@
 import org.eclipse.kura.core.net.util.NetworkUtil;
 import org.eclipse.kura.net.wifi.WifiAccessPoint;
 import org.eclipse.kura.net.wifi.WifiMode;
+import org.eclipse.kura.util.base.StringUtil;
```
Comment (marcellorinaldo) [inline kura/org.eclipse.kura.linux.net/src/main/java/org/eclipse/kura/linux/net/util/IWAPParser.java:24]: Update copyright header

File: kura/test/org.eclipse.kura.net.admin.test/src/test/java/org/eclipse/kura/net/admin/visitor/linux/WpaSupplicantConfigTest.java [hunk 5]
```diff
@@ -49,6 +49,7 @@
 import org.eclipse.kura.net.wifi.WifiMode;
 import org.eclipse.kura.net.wifi.WifiRadioMode;
 import org.eclipse.kura.net.wifi.WifiSecurity;
+import org.eclipse.kura.util.base.StringUtil;
```
Comment (marcellorinaldo) [inline kura/test/org.eclipse.kura.net.admin.test/src/test/java/org/eclipse/kura/net/admin/visitor/linux/WpaSupplicantConfigTest.java:52]: Update copyright header

File: kura/test/org.eclipse.kura.util.test/src/main/java/org/eclipse/kura/osgi/SingleServiceTrackerTest.java [hunk 6]
```diff
@@ -11,7 +11,7 @@
  *  Red Hat Inc
  *******************************************************************************/
 
-package org.eclipse.kura.utils;
+package org.eclipse.kura.osgi;
```
Comment (marcellorinaldo) [inline kura/test/org.eclipse.kura.util.test/src/main/java/org/eclipse/kura/osgi/SingleServiceTrackerTest.java:14]: Update copyright header

**Review by marcellorinaldo — APPROVED** [review]
Tested on Raspberry Pi 4. The PR seems good to me.

**Review by marcellorinaldo — APPROVED** [review]

**Pull request merged**